types/babar/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for babar 0.2
+// Project: https://github.com/stephan83/babar#readme
+// Definitions by: Matt Bachmann <https://github.com/Bachmann1234>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type color = 'yellow' | 'cyan' | 'white' | 'magenta' | 'green' | 'red' | 'grey' | 'blue' | 'ascii';
+
+interface Options {
+    caption?: string;
+    color?: color;
+    grid?: color;
```

Comment:
Sorry, I miss that 'ascii' is not a valid value for grid only for color

Reply:
I should be able to handle that.